creativecloud/blocks/nonprofit/nonprofit.js

```diff
@@ -159,14 +160,16 @@ async function sendOrganizationData() {
     }
 
     let body;
+    const { locale } = getConfig();
+    const ietf = locale.language;
```

Comment:
shouldn't it be locale.ietf instead of locale.language ?

Reply:
Thanks!